tests/setup_transaction_tests.py

```diff
@@ -103,26 +103,25 @@ def get_contract(source_code, *args, **kwargs):
 
 curve_order = 21888242871839275222246405745257275088548364400416034343698204186575808495617
 
+def get_logs(receipt, contract, event_name=None):
+    contract_log_ids = contract.translator.event_data.keys() # All the log ids contract has
```

Comment:
Why does `event_name` default to `None` when it's required in `get_log`?

Reply:
@DavidKnott @fubuloubu Hmm good point, considering this change. Can we rename get_log to get_last_log() and make `event_name=None` within `get_last_log` this way it's clear we want the last entry in the log (with optional even_name filtering).

Reply:
`event_name` is a filter, `None` means no filter applied
I like the suggestion about `get_last_log()` but that requires updates to the places it was used.